### ISSUE TITLE
[Bug] finetuning loss

### ISSUE THREAD

**Issue by linjing7** [hao-ai-lab/FastVideo#147]
### Environment

No

### Describe the bug

Hi, as described in the original paper, the model is trained with flow matching and computes the L1 loss of v. Why did this repo compute the loss on the latent code instead of v?

### Reproduction

No

**Comment by jzhang38** [comment]
We indeed trained on V. I don't think Hunyuan is trained with L1 loss?
<img width="781" alt="Screenshot 2025-01-15 at 1 47 03 PM" src="https://github.com/user-attachments/assets/ff68e62a-f689-4cab-9ea3-64db99be71d4" />

**Comment by linjing7** [comment]
Okay, thanks for your kind reply! Get it.
